extension-manifest-v3/scripts/download-engines/index.js

```diff
@@ -17,10 +17,23 @@ import shelljs from 'shelljs';
 import { ENGINE_VERSION, FiltersEngine } from '@cliqz/adblocker';
 import REGIONS from '../../src/utils/regions.js';
 
-function checksum(content) {
+function createChecksum(content) {
   return createHash('sha256').update(content).digest('hex');
 }
 
+function isChecksumMatched(path, checksum) {
+  if (existsSync(path) && createChecksum(readFileSync(path)) === checksum) {
+    if (process.stdout.isTTY) {
+      process.stdout.write('\r');
```

Comment:
what's the reason to avoid console.log and use `process.stdout`?

Reply:
Only then you can write in "the same line", the `console.log` adds `\n` as default. This is UI fix for logging downloaded engines - if they are fetched, then they are displayed one by one in lines, but if they are cached, they are displayed only during the check, and then next engine replaces the line. This is common pattern for building logs, to not overblow the console.